addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -332,6 +332,7 @@
 
                 courseJoinWidget.appendTo($widgetLocation);
                 if (!this.publicUser && courseJoinWidget.channel.channelEnroll === 'public' && this.slide.sessionAnswers) {
+                    this._applySessionAnswers();
```

Comment:
So in `start`, when coming back from creating an account, `isMember` is `false`, so it calls `renderJoinWidget` (where I added `_applySessionAnswers`) and eventually reloads the page. After reload `isMember` is `true`, so `_applySessionAnswers` is called, then `_submitQuiz`.
We could call `_applySessionAnswers` out of the `if`, so in case `sessionAnswers` is set, the answers are applied to the DOM. But calling `_applySessionAnswers` before `_renderJoinWidget` would set `sessionAnswers` as `false` and `_renderJoinWidget` needs `sessionAnswers` as true to call `joinChannel` and eventually set `isMember` as `true`.

Reply:
@ande-odoo So can't you call it after the call to `_renderJoinWidget` in start? I tried that and it seems to work ok but I may be missing an edge case

Seems more relevant to update it there rather than in a "render" method.
Cheers